src/views/EventPage.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+  <div>
+    <div class="column">
+      <div class="card ">
+        <section class="field fields title">
+          <h1>{{ event.get("name") }}</h1>
+          <button
+            class="primary"
+            @click="toggle(event)"
+            style="margin-left: 10px;"
+          >
+            {{ event.get("done") == true ? "Undone" : "Done" }}
+          </button>
+        </section>
+        <section class="field fields">
+          <h2>Details</h2>
+          <ul>
+            <li v-if="!editingTime">
+              Time:
+              {{
+                event.get("time").toLocaleString("en-US", {
+                  year: "numeric",
+                  month: "numeric",
+                  day: "numeric",
+                  hour: "numeric",
+                  minute: "numeric"
+                })
+              }}
+              <button
+                class="primary"
+                v-if="!editingTime"
+                @click="editingTime = true"
+                style="margin-left: 10px;"
+              >
+                Edit Time
+              </button>
+            </li>
+
+            <form v-if="editingTime" @submit.prevent="update(event)">
+              <h3>New Time</h3>
```

Comment:
Maybe load the current time into the field so that if the user just wanna make a minor change to the time they don't have to type everything again.